pkg/ccl/backupccl/testdata/backup-restore/import-start-time

```diff
@@ -0,0 +1,108 @@
+# Test that Import properly sets and removes the ImportStartTime from the table descriptor.
+# The basic idea:
+# For table with or without data:
+## -start and pause an import
+## -check that the ImportStartTime is set on the descriptor
+## -check that it's removed after cancellation / success
+
+new-server name=s1
+----
+
+exec-sql
+CREATE DATABASE d;
+USE d;
+CREATE TABLE foo (i INT PRIMARY KEY, s STRING);
+CREATE TABLE baz (i INT PRIMARY KEY, s STRING);
+INSERT INTO baz VALUES (1, 'x'),(2,'y'),(3,'z');
+----
+
+query-sql
+SELECT name, id FROM system.namespace WHERE name = 'foo';
+----
+foo 109
+
+exec-sql
+CREATE VIEW import_time (importStartTime)
+AS WITH tbls AS (
+   	SELECT id, crdb_internal.pb_to_json('cockroach.sql.sqlbase.Descriptor', descriptor) AS orig FROM system.descriptor
+   )
+   SELECT orig->'table'->'importStartWallTime' FROM tbls WHERE id = '109';
```

Comment:
How would you feel about a join to sys.namespace to get this ID by name instead of a literal 109 (and removing the query above that asserts the 109 result) ?

Reply:
TIL SQL :D

Done.